remove redundant apt command (#3590)
centreon/centreon-documentation: versioned_docs/version-24.04/reporting/installation.md

</TabItem>
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">

>   Install **gpg**:

Reviewer: ```suggestion
Install **gpg**:
```

  
```shell
apt install gpg
```

> Import the repo key:

Reviewer: ```suggestion
Import the repository key:
```

```shell
wget -O- https://apt-key.centreon.com | gpg --dearmor | tee /etc/apt/trusted.gpg.d/centreon.gpg > /dev/null 2>&1
```

> Add the following external repo (pour Java 8):

Reviewer: ```suggestion
Add the following external repository (for Java 8):
```

```shell
wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | apt-key add -
add-apt-repository --yes https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/
apt update
```

> Then install the Centreon MBI

Reviewer: ```suggestion
Then install Centreon MBI:
```

```shell

> apt install centreon-bi-server

Reviewer: ```suggestion
apt update && apt install centreon-bi-server
```
